atomic_reactor/plugins/pre_pin_operator_digest.py

```diff
@@ -196,7 +214,9 @@ def _get_pullspecs(self, operator_csv):
                    "automatically.".format(operator_csv.path))
             raise RuntimeError(msg)
         else:
-            self.log.warning("%s has a relatedImages section, skipping", operator_csv.path)
+            related_images = operator_csv._related_image_pullspecs()
+            self.log.info("%s has a relatedImages section, skipping", operator_csv.path)
```

Comment:
Maybe that `skipping` part should be removed, because we are processing pullspecs

Reply:
Removed.